docs/characterization/parallel_xeb.ipynb

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
<p>It would be good reference circuits/circuits_library directly here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3760/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>good idea, added!</p>